Web/starter_project_iconics/store/features/story/success-stories-api.ts

```diff
@@ -0,0 +1,16 @@
+import {createApi,fetchBaseQuery} from '@reduxjs/toolkit/query/react'
+import { SuccessStory } from '../../../types/story/success-stories'
+
+const API_BASE_URL = '/api';
+
+export const storyApi = createApi({
+    reducerPath: 'storiesApi',
+    baseQuery: fetchBaseQuery({baseUrl: API_BASE_URL}),
+    endpoints: (builder) => ({
+        getStories: builder.query<SuccessStory[], void>({
+            query: () => '/success-stories',
+        }),
+    }),
+})
+
+export const { useGetStoriesQuery} = storyApi
```

Comment:
I am not sure, but I don't think you need the features folder, just add the story folder inside the store